Public/Src/Pips/Dll/Operations/PipRemapWriter.cs

```diff
@@ -18,21 +19,28 @@ internal class PipRemapWriter : PipWriter
         private readonly PipDataEntriesPointerInlineWriter m_pipDataEntriesPointerInlineWriter;
         private readonly PipGraphFragmentContext m_pipGraphFragmentContext;
         private readonly PipExecutionContext m_pipExecutionContext;
+        private readonly char m_alternateSymbolSeparator;
+        private readonly char[] m_alternateSymbolSeparatorAsArray;
+        private readonly string m_alternateSymbolSeparatorAsString;
 
         /// <summary>
         /// Creates an instance of <see cref="PipRemapWriter"/>.
         /// </summary>
-        public PipRemapWriter(PipExecutionContext pipExecutionContext, PipGraphFragmentContext pipGraphFragmentContext, Stream stream, bool debug = false, bool leaveOpen = true, bool logStats = false)
-            : base(debug, stream, leaveOpen, logStats)
+        public PipRemapWriter(PipExecutionContext pipExecutionContext, PipGraphFragmentContext pipGraphFragmentContext, Stream stream, bool leaveOpen = true, char alternateSymbolSeparator = char.MinValue)
+            : base(debug: false, stream, leaveOpen, logStats: false)
         {
             Contract.Requires(pipExecutionContext != null);
             Contract.Requires(pipGraphFragmentContext != null);
             Contract.Requires(stream != null);
 
             m_pipExecutionContext = pipExecutionContext;
             m_pipGraphFragmentContext = pipGraphFragmentContext;
-            m_inliningWriter = new InliningWriter(stream, pipExecutionContext.PathTable, debug, leaveOpen, logStats);
-            m_pipDataEntriesPointerInlineWriter = new PipDataEntriesPointerInlineWriter(this, stream, pipExecutionContext.PathTable, debug, leaveOpen, logStats);
+            m_inliningWriter = new InliningWriter(stream, pipExecutionContext.PathTable, debug: false, leaveOpen, logStats: false);
+            m_pipDataEntriesPointerInlineWriter = new PipDataEntriesPointerInlineWriter(this, stream, pipExecutionContext.PathTable, debug: false, leaveOpen, logStats: false);
+
+            m_alternateSymbolSeparator = alternateSymbolSeparator;

```

Comment:
Instead of having `m_alternateSymbolSeparator` as a member variable, can we not just use `m_alternateSymbolSeparatorAsString`? For enablement, one can check if `m_alternateSymbolSeparatorAsString` is empty/not, and for writing, one can use `m_alternateSymbolSeparatorAsString[0]`.